ts/__e2e__/payment.e2e.ts

```diff
@@ -97,7 +97,7 @@ describe("Payment", () => {
       await element(by.text(I18n.t("global.navigator.wallet"))).tap();
       await element(by.text(I18n.t("wallet.payNotice"))).tap();
 
-      await element(by.text(I18n.t("wallet.QRtoPay.setManually"))).tap();
+      await element(by.text(I18n.t("barcodeScan.tabs.input"))).tap();
```

Comment:
What do you think about switching from `by.text` to `by.id` (and adding the `testId` property to the tab element)?

Reply:
Changed in https://github.com/pagopa/io-app/pull/5477/commits/0e1f72f44dc087abe3a8c82eb7fadc5179c32ca2